public/app/features/live/dashboard/dashboardWatcher.ts

```diff
@@ -11,14 +11,22 @@ import {
 } from '@grafana/data';
 import { getGrafanaLiveSrv, locationService } from '@grafana/runtime';
 import { appEvents, contextSrv } from 'app/core/core';
-import { sessionId } from 'app/features/live';
 
 import { ShowModalReactEvent } from '../../../types/events';
 import { getDashboardSrv } from '../../dashboard/services/DashboardSrv';
 
 import { DashboardChangedModal } from './DashboardChangedModal';
 import { DashboardEvent, DashboardEventAction } from './types';
 
+// sessionId is not a security-sensitive value.
+// It is used for filtering out dashboard edit events from the same browsing session
+const sessionId =
+  (window as any)?.grafanaBootData?.user?.id +
+  '/' +
+  Date.now().toString(16) +
+  '/' +
+  Math.random().toString(36).substring(2, 15);
```

Comment:
Maybe to avoid the warning

```suggestion
  uuidv4()
```

with:
```
import { v4 as uuidv4 } from 'uuid';
```

Reply:
👍  done